client/components/scene/scene.tsx

```diff
@@ -58,6 +58,14 @@ export default function Scene({ style }: Props): ReactElement {
     scene.setEulerAngles(rotation);
   }
 
+  function handleMouseWheel(e: WheelEvent<HTMLCanvasElement>) {
+    if (!sceneRef.current) return;
+    const scene = sceneRef.current;
+
+    const distance = scene.cameraDistance;
+    scene.cameraDistance = distance + (distance / e.deltaY) * 20;
+  }
+
   return (
     <div className={styles.canvasContainer} style={style}>
       <canvas
```

Comment:
What would a possible test for this look like on the Front-End? Checking to see if the WheelEvent response triggers as expected given certain circumstance?

Reply:
I guess I would need to mock the SceneRenderer, trigger a WheelEvent, and check that the camera distance gets moved in the expected direction. I will leave that for the Future (TM).